composer.json

```diff
@@ -224,5 +227,7 @@
     "bin": ["bin/rector"],
     "config": {
         "sort-packages": true
-    }
+    },
+    "minimum-stability": "dev",
```

Comment:
This is something we should definitely discuss in person during phpCE :man_facepalming: